libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFNetwork.h

```diff
@@ -45,18 +61,18 @@ typedef void (^SFDataResponseBlock) (NSData * _Nullable data, NSURLResponse * _N
 - (nonnull NSURLSessionDataTask *)sendRequest:(nonnull NSURLRequest *)urlRequest dataResponseBlock:(nullable SFDataResponseBlock)dataResponseBlock;
 
 /**
- * Returns the current NSURLSession instance being used.
+ * Sets a session configuration to be used for network requests in the Mobile SDK.
  *
- * @return NSURLSession instance.
+ * @param sessionConfig Session configuration to be used.
+ * @param isBackgroundSession YES - if it is a background session configuration, NO - otherwise.
  */
-- (nonnull NSURLSession *)activeSession;
++ (void)setSessionConfiguration:(nonnull NSURLSessionConfiguration *)sessionConfig isBackgroundSession:(BOOL)isBackgroundSession SFSDK_DEPRECATED(6.2, 7.0, "Use 'setSessionConfiguration:' instead.");
```

Comment:
Marked the existing API deprecated for removal in `7.0`. There is an equivalent new API below that doesn't take the `useBackground` argument.